### PR TITLE
Functional alexy

### PR DESCRIPTION
**Pull request by ayefimov-1**
Please review this change.   I added a new inv file that should be used specifically for the cvp 
 compact jobs.  The only difference from the default.inv is in the last line
 
https://github.com/infrawatch/functional-tests/compare/functional-alexy?expand=1#diff-81f746bf052c8428eedaa824c7fc262bdc4290d0b751239acb2ea1a0ac44140d

### CODE REVIEW

**Review by lnatapov — COMMENTED** [review]
This inv file is tailored for long lived STF on PSI. Note Line 52: 

ocp_cluster ansible_host="10.0.77.132" ansible_user="cloud-user" ansible_ssh_private_key_file="jumphost.key"

**Comment by ayefimov-1** [comment]
> This inv file is tailored for long lived STF on PSI. Note Line 52:
> 
> ocp_cluster ansible_host="10.0.77.132" ansible_user="cloud-user" ansible_ssh_private_key_file="jumphost.key"

@lnatapov this is intended to be long lived. The location of the jumphost.key file is slightly different in compact jobs.

**Review by lnatapov — COMMENTED** [review]
This is going to run only against 10.0.77.130. We have talked about variable there {{ocp_ip}}.

**Comment by ayefimov-1** [comment]
Final update to default-compact.inv file.

**Comment by elfiesmelfie** [comment]
I would advise not adding the new tasks to get_prom_info.yml

Before you run the functional tests, you should log into the cluster.

File: default-compact.inv [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# NOTE: for defining hosts both hostnames and IP addresses forms can be used
```
Comment (elfiesmelfie) [inline default-compact.inv:1]: All these lines are commented, so IMHO, this doesn't serve a purpose except as a sample inv file, in which case, the name should be changed

File: roles/client_side_tests/tasks/get_prom_info.yml [hunk 2]
```diff
@@ -1,4 +1,17 @@
 ---
+- name: "Get kube password"
+  ansible.builtin.shell:
+    cmd: |
```
Comment (elfiesmelfie) [inline roles/client_side_tests/tasks/get_prom_info.yml:4]: This is very specific to the installer used to provision openshift.

This won't work, for example, on a cluster created with CRC.

I'd prefer to keep the login part as a pre-task that the user is expected to perform before running the functional tests.

**Pull request closed without merging**